next.clayui.com/content/docs/css/components/navigation/navigation-bar.md

```diff
@@ -0,0 +1,312 @@
+---
+title: "Navigation Bar"
+description: "A navigation bar, is a horizontal bar that provides several access points to different parts of a system."
+---
+
+<div class="nav-toc">
+
+- [Light](#light)
+- [Secondary](#secondary)
+- [Using Buttons](#using-buttons)
+
+</div>
+
+<div class="alert alert-info">Check the <a href="https://liferay.design/lexicon">Lexicon</a> <a href="https://liferay.design/lexicon/core-components/navigation/horizontal-nav/">Navigation Bar Pattern</a> for a more in-depth look at the motivations and proper usage of this component.</div>
+
+<div class="alert alert-warning">
+    You will need to adjust the z-index of <code class="gatsby-code-text">.navbar-collapse</code> in the collapsed state if there are multiple <code class="gatsby-code-text">.navbar-collapse-absolute</code>'s near each other.
+</div>
+
+<div class="alert alert-warning">
+    Bootstrap 4 doesn't support Dropdown Menu's with Popper.js positioning inside Navbars. They align them manually via CSS classes. See <a href="http://localhost:8000/docs/css/components/dropdown.html#alignment">Dropdown Alignment</a>.
+</div>
+
+## Light
+
+<div class="sheet-example">
+    <nav class="navbar navbar-collapse-absolute navbar-expand-md navbar-underline navigation-bar navigation-bar-light">
```

Comment:
hey @diegonvs when I was looking at the examples the navigation bar is above the navigation menu of the site when you give the scroll.

Reply:
Solved because there isn't a navigation menu on next.clayui.com